src/services/category.js

```
@@ -0,0 +1,37 @@
+
+    return await Category.create({
+      name,
+      appearance: {
+        icon,
+        color
+      }
+    })
+  },
+
+  getCategoryNames: async (limit, skip, sortOptions, categories, name) => {
+    const query = {}
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)